pkg/monitor/cluster/podconditions.go

```diff
@@ -108,23 +108,29 @@ func (mon *Monitor) _emitPodContainerStatuses(ps *corev1.PodList) {
 				continue
 			}
 
-			mon.emitGauge("pod.containerstatuses", 1, map[string]string{
+			containerStatus := map[string]string{
 				"name":          p.Name,
 				"namespace":     p.Namespace,
 				"nodeName":      p.Spec.NodeName,
 				"containername": cs.Name,
 				"reason":        cs.State.Waiting.Reason,
-			})
+			}
+
+			if cs.LastTerminationState.Terminated != nil {
+				containerStatus["lastTerminatedState"] = cs.LastTerminationState.Terminated.Reason
```

Comment:
Can monitors and dashboards properly segregate data by this metric if it is not present in all emitted metrics? Would a default value help?